rites/roman1969/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@romcal/rite-roman1969",
```

Comment:
are we also releasing this as the general roman calendar?

Reply:
Not for now, at the moment it's only for internal use. But eventually in the future, it's something to consider.

Reply:
so, what's going to be releasing as the main general roman calendar? i don't think we want to break that pathway

Reply:
For now, it’s the `romcal` package that continues to be published (as well as all the `@romcal/calendar.*`). Nothing more.

To avoid breaking anything, the `romcal` package could become an alias for a future package `@romcal/rite-roman1969` (or vice versa). Or maybe we will do it differently. 
Currently, other types of rites are not yet supported, and I think we still need to think about how to organize (and publish) these packages properly (including all the `@romcal/calendar.*`). So indeed, for now, I’m focusing on preparing the codebase in the direction of multi-rites + greater modularity, without introducing any breaking changes.